addons/website/controllers/main.py

```diff
@@ -502,15 +502,6 @@ def make_scss_custo(self, url, values):
         request.env['web_editor.assets'].make_scss_customization(url, values)
         return True
 
-    @http.route(['/website/update_visitor_timezone'], type='json', auth="public", website=True)
```

Comment:
In theory we should not be deleting routes in a stable version, but I guess this one is ok, as it happens quickly after the release, and we're removing the JS code that calls it (and it should get picked up by assets recompilations quickly).
Users with a cached version of the JS assets may temporarily get a 404 in the background (assuming no visible error is raised if the route gives a 404 - worth a quick check)

Reply:
Indeed, It returns a 404 and there is nothing visible except this console log.